modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/Commons.java

```diff
@@ -478,7 +478,7 @@ public static <T> Predicate<T> negate(Predicate<T> p) {
      * @see org.apache.calcite.plan.RelTrait#apply(TargetMapping)
      * @see org.apache.calcite.rel.core.AggregateCall#transform(TargetMapping)
      */
-    public static Mappings.TargetMapping trimmingMapping(int sourceSize, ImmutableBitSet requiredElements) {
+    public static Mapping trimmingMapping(int sourceSize, ImmutableBitSet requiredElements) {
```

Comment:
`Mappings.apply(mapping, ImmutableBitSet)` accepts Mapping instead of `Mappings.TargetMapping`.